app/models/queen.rb

```diff
@@ -1,2 +1,7 @@
 class Queen < Piece
+
+  def valid_move?(x, y)
+    moved?(x,y) && (radial_move?(x,y) || horizontal_or_vertical_move?(x,y) || diagonal_move?(x,y))
```

Comment:
Is `radial_move` necessary here? Wouldn't this also be covered in a diagonal? If I'm reading correctly, we're evaling the x and y distance twice by calling both methods